### ISSUE TITLE
alpha2 error

### ISSUE THREAD

**Issue by FurnyGo** [Zailer43/FZMM-Mod#4]
```Minecraft has crashed!
net.fabricmc.loader.impl.FormattedException: java.lang.VerifyError: Bad type on operand stack
Exception Details:
  Location:
    net/minecraft/class_1799.handler$zkm001$getTooltip(Lnet/minecraft/class_1657;Lnet/minecraft/class_1836;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;)V @152: invokeinterface
  Reason:
    Type 'net/minecraft/class_2585' (current frame, stack[0]) is not assignable to 'net/minecraft/class_5250'
  Current Frame:
    bci: @152
    flags: { }
    locals: { 'net/minecraft/class_1799', 'net/minecraft/class_1657', 'net/minecraft/class_1836', 'org/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable', 'net/minecraft/class_2487', 'java/util/List', integer, 'net/minecraft/class_2585', 'java/util/List', integer }
    stack: { 'net/minecraft/class_2585', 'java/lang/String' }
  Bytecode:
    0000000: 2ab6 03a4 9901 232c b903 7e01 0099 011a
    0000010: 2ab6 0219 3a04 b206 ad9a 0010 1904 c700
    0000020: 0bbb 06ff 59b7 0700 bf19 04b6 049b b907
    0000030: 0301 00b9 0706 0100 3a05 1905 b907 0701
    0000040: 0036 06bb 0709 5913 070b b707 0c3a 072d
    0000050: b604 b1c0 0376 3a08 1908 1908 b907 0701
    0000060: 0004 64b9 0710 0200 5703 3609 1509 1506
    0000070: 9f00 6019 0719 0515 09b9 0712 0200 c002
    0000080: 0615 0915 0604 64a0 0009 1307 14a7 0006
    0000090: 1307 16ba 071b 0000 b907 1e02 0057 1509
    00000a0: 0670 05a0 0027 1908 1907 b206 96b2 0495
    00000b0: b606 9db9 0722 0200 b903 7902 0057 bb07
    00000c0: 0959 1307 14b7 070c 3a07 8409 01a7 ff9f
    00000d0: 1907 b907 2501 00b6 0726 9a00 1b19 0819
    00000e0: 07b2 0696 b204 95b6 069d b907 2202 00b9
    00000f0: 0379 0200 572a b601 4c3a 09bb 0709 5919
    0000100: 09b8 072b b707 0c3a 0a19 0819 0ab2 0696
    0000110: b203 92b6 069d b907 2202 00b9 0379 0200
    0000120: 572d 1908 b607 2eb1                    
  Stackmap Table:
    append_frame(@41,Object[#258])
    full_frame(@108,{Object[#2],Object[#396],Object[#891],Object[#1184],Object[#258],Object[#886],Integer,Object[#1801],Object[#886],Integer},{})
    full_frame(@144,{Object[#2],Object[#396],Object[#891],Object[#1184],Object[#258],Object[#886],Integer,Object[#1801],Object[#886],Integer},{Object[#1801],Object[#518]})
    full_frame(@147,{Object[#2],Object[#396],Object[#891],Object[#1184],Object[#258],Object[#886],Integer,Object[#1801],Object[#886],Integer},{Object[#1801],Object[#518],Object[#518]})
    same_frame(@202)
    same_frame(@208)
    same_frame(@245)
    full_frame(@295,{Object[#2],Object[#396],Object[#891],Object[#1184]},{})

	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:463)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.VerifyError: Bad type on operand stack
Exception Details:
  Location:
    net/minecraft/class_1799.handler$zkm001$getTooltip(Lnet/minecraft/class_1657;Lnet/minecraft/class_1836;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;)V @152: invokeinterface
  Reason:
    Type 'net/minecraft/class_2585' (current frame, stack[0]) is not assignable to 'net/minecraft/class_5250'
  Current Frame:
    bci: @152
    flags: { }
    locals: { 'net/minecraft/class_1799', 'net/minecraft/class_1657', 'net/minecraft/class_1836', 'org/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable', 'net/minecraft/class_2487', 'java/util/List', integer, 'net/minecraft/class_2585', 'java/util/List', integer }
    stack: { 'net/minecraft/class_2585', 'java/lang/String' }
  Bytecode:
    0000000: 2ab6 03a4 9901 232c b903 7e01 0099 011a
    0000010: 2ab6 0219 3a04 b206 ad9a 0010 1904 c700
    0000020: 0bbb 06ff 59b7 0700 bf19 04b6 049b b907
    0000030: 0301 00b9 0706 0100 3a05 1905 b907 0701
    0000040: 0036 06bb 0709 5913 070b b707 0c3a 072d
    0000050: b604 b1c0 0376 3a08 1908 1908 b907 0701
    0000060: 0004 64b9 0710 0200 5703 3609 1509 1506
    0000070: 9f00 6019 0719 0515 09b9 0712 0200 c002
    0000080: 0615 0915 0604 64a0 0009 1307 14a7 0006
    0000090: 1307 16ba 071b 0000 b907 1e02 0057 1509
    00000a0: 0670 05a0 0027 1908 1907 b206 96b2 0495
    00000b0: b606 9db9 0722 0200 b903 7902 0057 bb07
    00000c0: 0959 1307 14b7 070c 3a07 8409 01a7 ff9f
    00000d0: 1907 b907 2501 00b6 0726 9a00 1b19 0819
    00000e0: 07b2 0696 b204 95b6 069d b907 2202 00b9
    00000f0: 0379 0200 572a b601 4c3a 09bb 0709 5919
    0000100: 09b8 072b b707 0c3a 0a19 0819 0ab2 0696
    0000110: b203 92b6 069d b907 2202 00b9 0379 0200
    0000120: 572d 1908 b607 2eb1                    
  Stackmap Table:
    append_frame(@41,Object[#258])
    full_frame(@108,{Object[#2],Object[#396],Object[#891],Object[#1184],Object[#258],Object[#886],Integer,Object[#1801],Object[#886],Integer},{})
    full_frame(@144,{Object[#2],Object[#396],Object[#891],Object[#1184],Object[#258],Object[#886],Integer,Object[#1801],Object[#886],Integer},{Object[#1801],Object[#518]})
    full_frame(@147,{Object[#2],Object[#396],Object[#891],Object[#1184],Object[#258],Object[#886],Integer,Object[#1801],Object[#886],Integer},{Object[#1801],Object[#518],Object[#518]})
    same_frame(@202)
    same_frame(@208)
    same_frame(@245)
    full_frame(@295,{Object[#2],Object[#396],Object[#891],Object[#1184]},{})

	at net.minecraft.class_1761.<init>(class_1761.java:114)
	at net.minecraft.class_1761$1.<init>(class_1761.java:15)
	at net.minecraft.class_1761.<clinit>(class_1761.java:15)
	at net.minecraft.class_1802.<clinit>(class_1802.java:22)
	at net.minecraft.class_1792.method_7867(class_1792.java:74)
	at net.minecraft.class_2248.method_8389(class_2248.java:554)
	at net.minecraft.class_7143$class_7144.method_41583(class_7143.java:60)
	at net.minecraft.class_7143$class_7144.method_41581(class_7143.java:66)
	at net.minecraft.class_7143.method_41579(class_7143.java:199)
	at net.minecraft.class_5458.method_44104(class_5458.java:98)
	at net.minecraft.class_5458.method_30566(class_5458.java:105)
	at java.base/java.util.LinkedHashMap.forEach(LinkedHashMap.java:721)
	at net.minecraft.class_5458.<clinit>(class_5458.java:104)
	at net.minecraft.class_2378.<clinit>(class_2378.java:326)
	at meteordevelopment.meteorclient.systems.modules.Modules.<clinit>(Modules.java:65)
	at net.minecraft.class_128.handler$znm000$onAddStackTrace(class_128.java:524)
	at net.minecraft.class_128.method_555(class_128.java:80)
	at net.minecraft.class_128.method_568(class_128.java:134)
	at net.minecraft.class_128.method_24305(class_128.java:274)
	at net.minecraft.client.main.Main.method_44604(Main.java:159)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	... 2 more

**Comment by Zailer43** [comment]
The bug seems to be from the meteor client or maybe it's an incompatibility with FZMM Mod as they both seem to modify ItemStack.getTooltip()

If you are sure that the crash is from FZMM Mod, I would appreciate if you could pass me the steps to reproduce it since I have never used meteor client and I have not been able to reproduce it

**Comment by FurnyGo** [comment]
https://pastebin.com/7A0CFyWy
mods 
![image](https://user-images.githubusercontent.com/68079109/188209396-f6c4a8e4-97c4-4c20-a01e-a622bee07920.png)

**Comment by Zailer43** [comment]
ok, what happens is that you are using version 1.18.2 in 1.19.2, I have not released for 1.19.2

**Comment by Zailer43** [comment]
> ok, what happens is that you are using version 1.18.2 in 1.19.2, I have not released for 1.19.2

https://github.com/Zailer43/FZMM-Mod/releases/tag/0.1.0-ALPHA-3
